whitenoise/asgi.py

```diff
@@ -0,0 +1,74 @@
+class AsgiWhiteNoise:
```

Comment:
As you mentioned in your original post, the docs still need an update to show end-users how to configure whitenoise with ASGI.